collect-raw-metric-data-extension/collect_raw_metric_data_extension/extension.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from localstack import config
+from localstack.aws.handlers.metric_handler import MetricHandler
+from localstack.extensions.api import Extension, aws, http
+from localstack.http import Request
+
+
+class MyExtension(Extension):
+    name = "collect-raw-metric-data-extension"
+
+    def on_extension_load(self):
+        # enable the metric collection by setting the env
+        os.environ["LOCALSTACK_INTERNAL_TEST_COLLECT_METRIC"] = "1"
```

Comment:
Would love if we maybe had a proper modifiable config at some point that is accessible here in a typed way :thinking: 

But that's a task for future LocalStack :sunglasses: